meta-refkit/conf/distro/include/refkit-ci.inc

```diff
@@ -49,6 +49,11 @@ require conf/distro/include/refkit-development.inc
 REFKIT_VM_IMAGE_TYPES = "wic.xz wic.zip wic.bmap wic.xz.sha256sum"
 
 #
+# pre/post-build oe-selftests started by CI builder, whitespace-separated.
```

Comment:
Yes, I think it is good enough for now. Long-term I suspect that we might have a need to configure additional settings for oe-selftest, for example to make it more similar to the actual build. But that can wait.

If that becomes necessary, we could have a refkit-ci.inc and a refkit-selftest.inc where refkit-ci.inc includes refkit-selftest.inc and adds buildhistory. That's also a bit cleaner than the current "include refkit-development.inc" (which so far was an implementation detail of the distro and invisible to the build system).